ddp-workspace/projects/ddp-brain/src/app/components/dashboard/dashboard.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { BehaviorSubject } from 'rxjs';
+import { DashboardComponent } from './dashboard.component';
+import { ActivatedRoute, convertToParamMap } from '@angular/router';
+import { mockComponent } from 'ddp-sdk';
+
+describe('DashboardComponent', () => {
+    let fixture: ComponentFixture<DashboardComponent>;
+    let component: DashboardComponent;
+    let debugElement: DebugElement;
+    const paramMapSubject = new BehaviorSubject(convertToParamMap({userGuid: '123'}));
+
+    beforeEach(async() => {
+        const toolkitDashboard = mockComponent({ selector: 'toolkit-dashboard-redesigned', inputs: ['selectedUserGuid'] });
+        await TestBed.configureTestingModule({
+            imports: [NoopAnimationsModule],
```

Comment:
I believe this import is not necessary.

Reply:
well, I think there would be nothing wrong to have this module in unit tests by default since it's super handy and super small